frontend/app/helpers/spree/structured_data_helper.rb

```diff
@@ -36,7 +36,7 @@ def structured_product_hash(product)
     end
 
     def structured_images(product)
-      image = product.variant_images.first
+      image = product.master.images.first
```

Comment:
@KirillKayumov @pmilewski this won't work for every scenario, we need to use 

```suggestion
      image = product.default_variant.images.first
```